charts/postgres-operator/crds/postgresteams.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: postgresteams.acid.zalan.do
+  labels:
+    app.kubernetes.io/name: postgres-operator
+  annotations:
+    "helm.sh/hook": crd-install
+spec:
+  group: acid.zalan.do
+  names:
+    kind: PostgresTeam
+    listKind: PostgresTeamList
+    plural: postgresteams
+    singular: postgresteam
+    shortNames:
+    - pgteam
+  scope: Namespaced
```

Comment:
No, the operator watches all namespaces for team crds and merges them into one big team configuration. Since namespaces do not necessarily correlate with teams or administrative domains that is fine. Aim is to have a convenient team setup.

Reply:
https://github.com/zalando/postgres-operator/blob/61225998ff6227bc9d58f97e8469ccc5ad6f6739/pkg/controller/util.go#L403